### PR TITLE
Classcontent parameters update

### PR DESCRIPTION
**Pull request by crouillon**
- Adding classcontent.postload event to be able to priorize listeners
- Introducing MalformedParameterException to prevent parameters without `value` entry

### CODE REVIEW

File: ClassContent/AbstractClassContent.php [hunk 1]
```diff
@@ -997,7 +1009,7 @@ public function getParam($key)
 
         $params = $this->defaultParams[$key];
 
-        if ($value !== null) {
+        if (is_array($value) && isset($value['value'])) {
```
Comment (eric-chau) [inline ClassContent/AbstractClassContent.php]: nice catch :+1:

File: ClassContent/AbstractClassContent.php [hunk 2]
```diff
@@ -1031,7 +1043,7 @@ public function getAllParams()
 
         foreach ($this->defaultParams as $key => $value) {
             $params[$key] = $value;
-            if (isset($instanceParams[$key])) {
+            if (isset($instanceParams[$key]) && isset($instanceParams[$key]['value'])) {
```
Comment (eric-chau) [inline ClassContent/AbstractClassContent.php]: this condition can be simplified to: ``if (isset($instanceParams[$key]['value'])) {``

File: Controller/Controller.php [hunk 3]
```diff
@@ -133,6 +133,10 @@ public function getEntityManager()
      */
     public function createFormBuilder($data)
     {
+        if (!class_exists('Symfony\\Component\\Form\\Forms')) {
```
Comment (eric-chau) [inline Controller/Controller.php]: Hm I think a rebase is missing? Cause this is not supposed be part of your PR.

File: Rest/Controller/KeywordController.php [hunk 4]
```diff
@@ -21,7 +21,7 @@
  * @author Charles Rouillon <charles.rouillon@lp-digital.fr>
  */
 
-namespace backbee\Rest\Controller;
+namespace BackBee\Rest\Controller;
```
Comment (eric-chau) [inline Rest/Controller/KeywordController.php]: Same for this

File: Controller/Controller.php (hunk 3 above)
Comment (mickaelandrieu) [inline Controller/Controller.php]: true :)

File: composer.json [hunk 5]
```diff
@@ -84,6 +84,9 @@
         "backbee/bb-core-js": "1.0.0",
         "symfony/phpunit-bridge": "~2.7.0"
     },
+    "suggest": {
```
Comment (eric-chau) [inline composer.json]: Same remark for this, must be not part of current PR.

**Comment by eric-chau** [comment]
Good job @crouillon, thank you :+1:

File: Controller/Controller.php (hunk 3 above)
Comment (crouillon) [inline Controller/Controller.php]: yes but it's rebased, i don't know why this commit appears

**Comment by mickaelandrieu** [comment]
:+1:  thank you @crouillon

**Pull request merged**